state/indexer/block/kv/kv.go

```diff
@@ -82,10 +87,136 @@ func (idx *BlockerIndexer) Index(bh types.EventDataNewBlockEvents) error {
 	if err := idx.indexEvents(batch, bh.Events, height); err != nil {
 		return fmt.Errorf("failed to index FinalizeBlock events: %w", err)
 	}
-
 	return batch.WriteSync()
 }
 
+func getKeys(indexer BlockerIndexer) [][]byte {
```

Comment:
Minor. Why isn't this a method of `BlockerIndexer`? Something like:
`func (indexer BlockerIndexer) getKeys() [][]byte {`

Reply:
Also, this function is only called from test code. Consider moving it to test code.